lib/components/ColorPicker.js

```diff
@@ -41,12 +42,15 @@ const ColorPicker = props => {
     setColorPickerDisplay(false);
   };
   const handleChange = color => {
-    if (!color.hex) {
-      setColor({ hex: initialColor });
-      return;
-    }
-    setColor(color);
-    onChange(color, title);
+    clearTimeout(timer);
+    timer = setTimeout(() => {
```

Comment:
Few questions to this code.
1. Why we need setTimeout and pushing `setColor` to end of event loop ?
2. Is `500` some kind of magic number here :) ?

Reply:
debounce removed, it's not necessary here